### ISSUE TITLE
Crash at startup (Fabric 1.17.1)

### ISSUE THREAD

**Issue by maicol07** [squeek502/AppleSkin#135]
Error:

```
Cannot read field "maxHudOverlayFlashAlpha" because "squeek.appleskin.ModConfig.INSTANCE" is null
```

Crash report: [bytebin.lucko.me/eXGbl5rsGT](https://bytebin.lucko.me/eXGbl5rsGT)

Crash log: [bytebin.lucko.me/Mml6fju7qD](https://bytebin.lucko.me/Mml6fju7qD)

Similar to #118

**Comment by squeek502** [comment]
Thanks for the report. Looks like a null check in `HUDOverlayHandler.onClientTick` is necessary.

**Comment by squeek502** [comment]
Should be fixed in v2.1.3
